src/Proto.Cluster/Cluster.cs

```diff
@@ -81,9 +81,10 @@ public Cluster(ActorSystem system, ClusterConfig config)
 
         public async Task StartMemberAsync()
         {
+            //gossiper must be started whenever any topology events starts flowing
+            await Gossip.StartAsync();
```

Comment:
This was one of the bugs. the gossip actor ofc needs to be started before registration to process events